src/utils/updateIdentity.ts

```diff
@@ -7,10 +7,10 @@ type UpdateBody = {
   value: string;
```

Comment:
This should go inside the business component.

Reply:
OK done 0733f6a